reference/orders.v2.oas2.yml

```diff
@@ -632,9 +632,9 @@ paths:
 
         There are three methods for generating a tracking link for a shipment:
 
-        1. Use `shipping_provider` and `tracking_number`: This generates an automatic tracking link that you can click from the BigCommerce control panel and customer-facing emails. The `generated_tracking_link` property in the API response represents this tracking link. The `tracking_link` property in the API response will remain empty.
+        1. Use `shipping_provider` and `tracking_number`: This generates an automatic Aftership tracking link that you can click from the BigCommerce control panel and customer-facing emails. The `generated_tracking_link` property in the API response represents this tracking link. The `tracking_link` property in the API response will remain empty.
```

Comment:
Aftership is not always used.
If we have a natively integrated provider we provide a link to that provider.
For instance if a merchant is using Fedex and has it setup in shipping as a provider, it will provide a Fedex link if fedex is sent, if they have not setup Fedex, it will provide an aftership link.